src/body.js

```diff
@@ -5,13 +5,14 @@
  * Body interface provides common methods for Request and Response
  */
 
-import {convert} from 'encoding';
 import Stream, {PassThrough} from 'stream';
 import Blob, {BUFFER} from './blob.js';
 import FetchError from './fetch-error.js';
 
 const DISTURBED = Symbol('disturbed');
 
+try { var convert = require('encoding').convert } catch(e) {}
```

Comment:
Please no implicit `var`... declare `convert` outside the `try`. (Also, use `let`.)